test/run_envoy_bazel_coverage.sh

```diff
@@ -36,7 +36,10 @@ mkdir -p "${COVERAGE_DIR}"
 
 COVERAGE_IGNORE_REGEX="(/external/|pb\.(validate\.)?(h|cc)|/chromium_url/|/test/|/tmp)"
 COVERAGE_BINARY="bazel-bin/test/coverage/coverage_tests"
-COVERAGE_DATA="bazel-out/k8-fastbuild/testlogs/test/coverage/coverage_tests/coverage.dat"
+COVERAGE_DATA="${COVERAGE_DIR}/coverage.dat"
+
+echo "Merging coverage data..."
+llvm-profdata merge -sparse -o ${COVERAGE_DATA} $(find -L bazel-out/k8-fastbuild/testlogs/test/coverage/coverage_tests/ -name coverage.dat)
```

Comment:
Do you know how well this works when rebuilding locally after making a change and not cleaning the Bazel cache?

Reply:
I'm not aware of any issue, coverage tests are no longer run out of sandbox and bazel is aware of the coverage.dat.

Reply:
Great, being back in the sandbox is a huge win when reasoning about this kind of thing.